Sources/TuistCore/Models/Scheme.swift

```diff
@@ -153,6 +153,24 @@ public class TestAction: Equatable {
     }
 }
 
+public struct TestableTarget: Equatable, Hashable, ExpressibleByStringLiteral {
+    public let target: String
+    public let isSkipped: Bool
+    public let isParallelizable: Bool
+    public let isRandomExecutionOrdering: Bool
+
+    public init(target: String, skipped: Bool = false, parallelizable: Bool = false, randomExecutionOrdering: Bool = false) {
+        self.target = target
+        self.isSkipped = skipped
+        self.isParallelizable = parallelizable
+        self.isRandomExecutionOrdering = randomExecutionOrdering
+    }
+
+    public init(stringLiteral value: String) {
```

Comment:
At the `TuistCore` level the `ExpressibleByStringLiteral` conformance is not needed - it's only needed in the `ProjectDescription` for convenience and backwards compatibility.